cli/src/fmt/simple/asset.rs

```diff
@@ -1,21 +1,38 @@
-use fmt::Render;
+use fmt::{indent, Render};
 use super::Simple;
-use stellar_client::resources::Asset;
+use stellar_client::resources::{Asset, AssetIdentifier, Flags};
 
 impl Render<Asset> for Simple {
     fn render(&self, asset: &Asset) -> Option<String> {
         let mut buf = String::new();
-        append_to_buffer!(buf, "Code:         {}", asset.code());
-        append_to_buffer!(buf, "Type:         {}", asset.asset_type());
-        append_to_buffer!(buf, "Issuer:       {}", asset.issuer());
+        append_to_buffer!(buf, "Asset:");
+        nest_in_buffer!(self, buf, asset.identifier());
         append_to_buffer!(buf, "Amount:       {}", asset.amount());
-        append_to_buffer!(buf, "Num accounts: {}", asset.num_accounts());
+        append_to_buffer!(buf, "Num Accounts: {}", asset.num_accounts());
         append_to_buffer!(buf, "Flags:");
-        if asset.is_auth_required() {
-            append_to_buffer!(buf, "  auth is required");
+        nest_in_buffer!(self, buf, &asset.flags());
+        Some(buf)
+    }
+}
+
+impl Render<AssetIdentifier> for Simple {
+    fn render(&self, asset_id: &AssetIdentifier) -> Option<String> {
+        let mut buf = String::new();
+        append_to_buffer!(buf, "Type:   {}", asset_id.asset_type());
```

Comment:
In another place I just made this `<code>-<issuer` unless native, and then it's just `XLM`. Can we do that here as well?

Reply:
If we want just a one-liner output, should this implement `fmt::Display` rather than `Render`?